lib/config/presets/bitbucket-server/index.spec.ts

```diff
@@ -88,7 +88,7 @@ describe('config/presets/bitbucket-server/index', () => {
       ).rejects.toThrow(PRESET_DEP_NOT_FOUND);
     });
 
-    it('throws to big', async () => {
+    it('throws too big', async () => {
```

Comment:
I'm not sure if `fix(bitbucket-server): fix typo in log message and spec file` is the correct PR title? Feel free to change it to the correct title. 😉